packages/odyssey-storybook/src/components/odyssey-mui/Toast/Toast.stories.tsx

```diff
@@ -10,64 +10,61 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import { Meta, Story } from "@storybook/react";
+import { Button, Toast, ToastProps, ToastStack } from "@okta/odyssey-react-mui";
 import * as React from "react";
-import { Story } from "@storybook/react";
-import {
-  Alert,
-  AlertTitle,
-  Button,
-  CloseIcon,
-  Link,
-  Snackbar,
-  Stack,
-} from "@okta/odyssey-react-mui";
-import { MuiThemeDecorator } from "../../../../.storybook/components";
 
 import ToastMdx from "./Toast.mdx";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+import { useState } from "react";
 
-export default {
+const storybookMeta: Meta<ToastProps> = {
   title: `MUI Components/Alerts/Toast`,
-  component: Alert,
+  component: Toast,
   parameters: {
     docs: {
       page: ToastMdx,
     },
   },
   argTypes: {
-    actionLink: {
-      control: "text",
-      default: null,
+    autoHideDuration: {
+      control: "number",
+      defaultValue: null,
+    },
+    isDismissable: {
+      control: "boolean",
     },
-    content: {
+    linkText: {
       control: "text",
-      defaultValue: "Mission to Sirius B scheduled for January 7, 2023",
     },
-    isDismissible: {
-      control: "boolean",
-      defaultValue: null,
+    linkUrl: {
+      control: "text",
     },
     role: {
       control: "radio",
-      options: ["status", null],
+      options: ["alert", "status", null],
```

Comment:
This component doesn't explicitly allow for `null`, so we should change it to `undefined` like the type says.